frontend/cypress/integration/nurse-scheduling-examples/example.spec.js

```diff
@@ -0,0 +1,15 @@
+/// <reference types="cypress" />
+
+context("Example", () => {
+  beforeEach(() => {
+    cy.visit("localhost:3002");
```

Comment:
A to nie powinno brać z jakiejś zmiennej środowiskowej? Bo ja to mam na porcie 3000

Reply:
Też mam na tym porcie